twake/frontend/src/app/services/RouterService.ts

```diff
@@ -257,6 +257,14 @@ class RouterServices extends Observable {
     }
     return false;
   }
+
+  pushTo(path: string, state?: unknown): void {
```

Comment:
This method name does not feel natural, why don't we keep `push`?

Reply:
Renamed to `push`